template/.github/ISSUE_TEMPLATE/publish_component.md

```diff
@@ -0,0 +1,50 @@
+---
+name: Publish Component
+about: Create a ticket to drive compliance before publishing a a new componet to Auro Design System
+title: 'auro-[name]; publish to Auro Design System'
+labels: 'Status: Proposal, not-reviewed'
+---
+
+# Publish Component
+
+## Specific steps required to publish this component.
+
+All of the below outlined steps must be completed.
+
+### Design & Planning
+- [ ] Reviewed [AuroLabs documentation](http://auro.alaskaair.com/)
+- [ ] Reviewed [Auro's Contributing Guidelines](http://auro.alaskaair.com/contributing)
+- [ ] Reviewed [Auro's CSS conventions](http://auro.alaskaair.com/webcorestylesheets/conventions) and how to apply CSS to custom elements
+- [ ] Design team engaged in designing the new component
+- [ ] Design work integrated in the Auro Design System UI Kit
+
+### Development
+- [ ] Component built using latest version of [WC-Generator](http://auro.alaskaair.com/getting-started/developers/generator/install)
+- [ ] Addresses all design and UX as proposed in the design comp
+- [ ] A11y in regards to role and live notifications need to be coded into the final WC
+- [ ] All style code follows [Auro's Style Guide](http://auro.alaskaair.com/webcorestylesheets/guidelines)
+
+### Documentation
+- [ ] Demo page created for the Auro Doc Site
+- [ ] Examples and use case info
+- [ ] Bundled version info
+- [ ] README doc completed with all relative information
+- [ ] API documentation created
+
+### Testing
+- [Browser compatibility](http://auro.alaskaair.com/support/browsersSupport) - desktop and mobile versions
+  - [ ] Chrome
+  - [ ] Firefox
+  - [ ] Edge (79 +)
+  - [ ] Safari
+- [Accessibility](http://auro.alaskaair.com/a11y-statement)
+  - [ ] Axe / Google Lighthouse (baseline standard)
+  - [ ] Screenreader (determine usability of experience via screenreader)
+  - [ ] Keyboard use (a user must be able to navigate to key parts via a keyboard)
+  - [ ] Accessibility should be verified against all browsers
+- [ ] Bundled versions tested
+
+### Review
+- [ ] Review demo with the designer as well as the Auro team
+- [ ] Review demo with Auro Team
```

Comment:
Don't think we need this one?

Reply:
My thinking is that everything before the "review" section is largely done without Auro Team involvement (probably some of course). An experienced Auro contributor shouldn't need much from us up until it's time to review.

The "review" section is really where we as the Auro Design System team are giving our final once over. While we may not need to check these off ourselves, I included them more as a complete list of everything the contributor will need to understand as "gate-keeper" tasks before becoming an official Auro component.

Thoughts?

Reply:
I can get on board with that, but we then need to clearly communicate that. Like some kind of instruction. From me reading this list, I didn't get that context.

Reply:
I've updated the language in the template to reflect this.

Reply:
I am not seeing updated content?

Reply:
Should be there now.